clang/lib/AST/JSONNodeDumper.cpp

```diff
@@ -1579,11 +1579,11 @@ void JSONNodeDumper::VisitMaterializeTemporaryExpr(
 }
 
 void JSONNodeDumper::VisitCXXDefaultArgExpr(const CXXDefaultArgExpr *Node) {
-  attributeOnlyIfTrue("hasRewrittenInit", Node->hasRewrittenInit());
+  JOS.attribute("hasRewrittenInit", Node->hasRewrittenInit());
```

Comment:
So if IIUC this used to only write the attribute if the value was `true`, so we should have a test for both cases `true` and `false` to verify this is working as intended.